README.md

```diff
@@ -16,7 +16,7 @@ and [here](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/d
 
 ## Quickstart
 
-Run local Data Explorer with a test dataset:
+Run local Data Explorer with the 1000 Genomes dataset:
```

Comment:
This comment is actually for .gitignore

- Remove lines 4-6
- Have .gitignore ignore 1000_genomes/deploy.json. In general, deploy.json shouldn't be checked in. People can deploy to whatever project they want.

Also, in dataset_config/1000_genomes/README.md, remove "See here for other...".

Reply:
Done.